FluidNC/src/WebUI/WebSettings.cpp

```diff
@@ -633,6 +659,10 @@ namespace WebUI {
     }
 
     static Error renameObject(const char* fs, char* parameter, AuthenticationLevel auth_level, Channel& out) {
+        if (fs == sdName && !config->_sdCard->is_configured(out)) {
+            return Error::InvalidStatement;
+        }
+
```

Comment:
Another reason for putting the check in FluidPath is that the fs argument to renameObject and friends is just the default, which can be overridden with an explicit prefix like /localfs/ or /sd/ .  Thus checking based on fs == sdName will fail to catch some uses of SD, and will wrongly complain if the sd default is overridden via an explicit /localfs/ prefix in the parameter.